packages/expo-dev-launcher/bundle/functions/__tests__/getDevSessionsAsync.test.ts

```diff
@@ -0,0 +1,46 @@
+import { mockFetchReturn } from '../../test-utils';
+import { getDevSessionsAsync } from '../getDevSessionsAsync';
+
+const mockFns = [fetch] as jest.Mock[];
+
+describe('getDevSessionsAsync()', () => {
+  beforeEach(() => {
+    mockFns.forEach((fn) => fn.mockClear());
+  });
+
+  test('authenticated call', async () => {
+    await getDevSessionsAsync({ isAuthenticated: true, installationID: '321', isDevice: true });
+
+    expect(fetch).toHaveBeenCalledWith(
+      expect.stringContaining('/development-sessions'),
+      expect.any(Object)
+    );
+  });
+
+  test('does not use installationI if sessions are found via authenticated request', async () => {
```

Comment:
maybe add a test case for if `isAuthenticated` is true but no sessions are found (in which case i think there should be a second request that _does_ use installation ID)